packages/wonder-blocks-dropdown/src/components/dropdown-core.js

```diff
@@ -699,15 +695,26 @@ class DropdownCore extends React.Component<Props, State> {
                     placement={
                         alignment === "left" ? "bottom-start" : "bottom-end"
                     }
-                    modifiers={{
-                        wbVisibility: visibilityModifierDefaultConfig,
-                        preventOverflow: {
-                            boundariesElement: "viewport",
-                            escapeWithReference: true,
+                    modifiers={[
+                        {
+                            name: "preventOverflow",
+                            options: {
+                                rootBoundary: "viewport",
+                                // Allows to overlap the popper in case there's
+                                // no more vertical room in the viewport.
+                                altAxis: true,
```

Comment:
This is the line that fixes WB-962. This is possible only with v2, and that's why we had to do the rest of the work involved on this PR 😅.

Reply:
Wow, so much work!